### ISSUE TITLE
Default value for optional parameters is not shown in help message.

### ISSUE THREAD

**Issue by AgarwalPragy** [moigagoo/cliar#3]
Optional arguments show as follows:
```
optional arguments:
  --trials TRIALS    Maximum trials for each URL in case the API returns an error.
```

This leaves the user with no information about the default value. One can manually include the default value in the help text, but that results in duplicate code and forces the developer to update the value in two places should the default value change.

It would be nice if instead the default value could show up in a new line, as:

```
optional arguments:
  --trials TRIALS    Maximum trials for each URL in case the API returns an error.
                     default: 5
```

**Comment by moigagoo** [comment]
Thanks for creating the issue! I've just pushed the change that would allow it: https://github.com/moigagoo/cliar/pull/5

**Comment by moigagoo** [comment]
Fixed in 1.2.4: https://moigagoo.github.io/cliar/changelog/#124
